Crash/LocalSpeck.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Rhino.FileIO;
+using Rhino.Geometry;
+using Rhino.Runtime;
+using SpeckLib;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Crash
+{
+
+    /// <summary>
+    /// Local instance of a received speck.
+    /// </summary>
+    public sealed class LocalSpeck : Speck
```

Comment:
Instead of inheriting from Speck and having to copy all of its properties, how about something like:
```
public sealed class LocalSpeck
{
  public Speck Speck { get; set; }
  public GeometryBase Geometry { get; set; }
}
```

Reply:
This seems like a much better idea really

Reply:
I did something a little different. I've created the ISpeck interface like before, and implemented it to point at the speck inside it